css/src/components/hero.scss

```diff
@@ -25,8 +25,10 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 	align-items: stretch;
 	justify-content: space-between;
 
-	@include tablet {
-		flex-direction: row;
+	&:has(.hero-details) {
```

Comment:
This would mark the first use of `:has` in Atlas (which I personally find exciting)

It's worth calling out that [support for `:has`](https://caniuse.com/css-has) is at ~92% adoption right now. That's pretty good - but it maybe could be better for something as prominent as a hero?

I wanted to volunteer a possible change, to see what we think of it:
1. Remove this `flex-direction: row` declaration altogether.
2. ...and then use our templates and scripts to apply the [`flex-direction-row-tablet` atomic](https://design.learn.microsoft.com/atomics/flex.html) to heroes with detail cards.

Thoughts?

Reply:
I, on the other hand, like Aaron's approach here. Let the CSS handle styling the hero elements instead of our templates/scripts. Thoughts?

Reply:
what about just moving this rule to line 55?

Reply:
@ollips I don't think that would work, since that would apply to `.hero-details`, which is a child of the flex container.

Reply:
I see it now! Yes, this makes sense.